.github/scripts/artifact-upload-params.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+case "$ARTIFACT_SELECTOR" in
+  peer)
+    ARTIFACT_PATH='target/release/humanode-peer'
+    ARTIFACT_NAME="humanode-peer-$(rustc -vV | sed -n 's|host: ||p')"
+
+    if [[ "$PLATFORM_ARTIFACT_MARKER" != "" ]]; then
+      ARTIFACT_NAME="${ARTIFACT_NAME}-${PLATFORM_ARTIFACT_MARKER}"
+    fi
```

Comment:
I'm thinking this section and the `MODE_ARTIFACT_MARKER` should be applied universally for all artifact selectors

Reply:
We could make it changeable in the mode - but let's leave it for now, it is good enough